sbt

```diff
@@ -229,8 +231,13 @@ execRunner () {
     vlog ""
   }
 
-  [[ -n "$batch" ]] && exec </dev/null
-  exec "$@"
+  setTrapExit
+
+  if [[ -n "$batch" ]]; then
+    "$@" < /dev/null
+  else
+    "$@"
```

Comment:
It's required in the sense that I didn't get echo back until I removed it. Maybe there's another way, but that's just the investigation I sought to avoid.